src/components/play-bar/desktop/PlayBar.js

```diff
@@ -321,6 +320,9 @@ class PlayBar extends Component {
       playButtonStatus = 'play'
     }
 
+    const favoriteAndRepostIsDisabled = !uid || isOwner
```

Comment:
i dont think we have a true standard as a team yet, but imo starting booleans with `is` makes it easier to identify, so `isFavoriteAndRepostDisabled`. what do you think?

Reply:
good catch, this is a standard! https://www.notion.so/audiusproject/Client-Javascript-Styleguide-4f34640139fc4a809c3765131bc597ff#d99d63f262f24990a83665bedd3b8610

Reply:
100% we should standardize around the verbs 

isReposted
hasUserFavorited
wenMoon